.github/component_owners.yml

```
@@ -17,67 +21,101 @@ components:
     - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - carolabadeer
+  packages/opentelemetry-propagation-utils: []
+    # Unmaintained?
   packages/opentelemetry-redis-common:
     - haddasbronfman
+  packages/opentelemetry-test-utils: []
+    # Unmaintained?
   plugins/node/instrumentation-amqplib:
     - blumamir
   plugins/node/instrumentation-dataloader:
     - henrinormak
   plugins/node/instrumentation-fs:
     - rauno56
+  plugins/node/instrumentation-lru-memoizer:
+    - blumamir
+  plugins/node/instrumentation-mongoose:
+    - blumamir
+  plugins/node/instrumentation-socket.io:
+    - mottibec
+    - rauno56
   plugins/node/instrumentation-tedious:
     - rauno56
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - carolabadeer
   plugins/node/opentelemetry-instrumentation-aws-sdk:
     - carolabadeer
     - blumamir
+  plugins/node/opentelemetry-instrumentation-bunyan: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-cassandra: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-connect: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-dns: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-express: []
+    # Unmaintained?
```

Originally contributed by @vmarchaud  

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-express:
    - vmarchaud
```

---

I don't think that would be a good idea, folks at Aspeco have their own fork/version (https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/instrumentation-express) maybe @blumamir might be interested ?